src/Domain/DomainCreators/DomainCreator.hpp

```diff
@@ -56,13 +56,14 @@ struct domain_creators<2> {
 };
 template <>
 struct domain_creators<3> {
-  using type = typelist<
-      DomainCreators::Brick  //, DomainCreators::RubiksCubeWithHole,
-                             //                DomainCreators::Shell,
-                             //                DomainCreators::Sphere,
-                             //                DomainCreators::RotatedBricks,
-                             //                DomainCreators::UnevenBricks
-      >;
+  using type =
+      typelist<DomainCreators::Brick  //, DomainCreators::RubiksCubeWithHole,
+                                      //                DomainCreators::Shell,
+               ,
```

Comment:
Choose some sane way to format this.

Reply:
Please just remove the unused classes, and add them back later (if they are even ported)